qa/setup_packages.py

```diff
@@ -325,7 +325,7 @@ def get_pyvers_name(self, url, cuda_version):
                         "mxnet-cu{cuda_v}"),
                 CudaPackage("tensorflow-gpu",
                         { "90"  : [PckgVer("1.12.0", python_max_ver="3.7")],
-                          "100" : [PckgVer("1.15.2",  python_max_ver="3.7"), PckgVer("2.1.0",  python_max_ver="3.7"), \
+                          "100" : [PckgVer("1.15.3",  python_max_ver="3.7"), PckgVer("2.2.0",  python_max_ver="3.7"), \
```

Comment:
```suggestion
                          "100" : [PckgVer("1.15.3",  python_max_ver="3.7"), PckgVer("2.1.0",  python_max_ver="3.7"), \
```
We already have 2.2 supported. Check the line bellow.